readme.md

```diff
@@ -802,6 +802,7 @@ I think of these stories often; The Death of Dr. Island won a Nebula and offers
 #### [2001: A Space Odyssey](http://www.imdb.com/title/tt0062622/) (1968) [8.3]
 
 Director: _Stanley Kubrick_
```

Comment:
You need to add two blank spaces on the end of this to force a linebreak. Lol Markdown.

This applies to everywhere `Categories` is together with `Director`.